py/obj.h

```diff
@@ -728,7 +728,7 @@ mp_obj_t mp_obj_new_memoryview(byte typecode, size_t nitems, void *items);
 const mp_obj_type_t *mp_obj_get_type(mp_const_obj_t o_in);
 const char *mp_obj_get_type_str(mp_const_obj_t o_in);
 bool mp_obj_is_subclass_fast(mp_const_obj_t object, mp_const_obj_t classinfo); // arguments should be type objects
-mp_obj_t mp_instance_cast_to_native_base(mp_const_obj_t self_in, mp_const_obj_t native_type);
+mp_obj_t mp_cast_to_native_base(mp_obj_t self_in, mp_const_obj_t native_type);
```

Comment:
Do you think `mp_obj_cast_to_native_base()` is a better name, to be consistent with the other `mp_obj_` names here?

Reply:
Definitely. Updated.